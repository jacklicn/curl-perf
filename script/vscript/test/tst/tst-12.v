

var s = "The quick brown fox jumps over the lazy dog", f = "fox", g = "frog"


print( index(s,f) )


print( index(s,g) )


print( rindex(s,f) )


print( rindex(s,g) )


print( substr(s,16,3) )

print( substr(s,16) )

print( length(s) )

print( length(f) )

print( length(1212) )