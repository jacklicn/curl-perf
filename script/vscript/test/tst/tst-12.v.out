*** syntax ok ***
*** semantics and code generation ok ***

>16

>-1

>16

>-1

>fox

>fox jumps over the lazy dog

>43

>3

>4
